lib/qs/test_runner.rb

```diff
@@ -40,4 +36,41 @@ def normalize_params(params)
 
   end
 
+  class JobTestRunner < TestRunner
+
+    def initialize(handler_class, args = nil)
+      if !handler_class.include?(Qs::JobHandler)
+        raise InvalidJobHandlerError, "#{handler_class.inspect} is not a"\
+                                      " Qs::JobHandler"
+      end
+
+      super(handler_class, args)
+    end
+
+  end
+
+  class EventTestRunner < TestRunner
```

Comment:
@jcredding we discussed you *could* just rename test runner as "job test runner and have the event test runner subclass.  That job handler check shouldn't fail on event handlers since they bring job handler in.